docker/Makefile

```diff
@@ -71,12 +71,21 @@ build_% : repository
 		--tag pdo_$*:$(PDO_VERSION) \
 		--file '$(DOCKER_DIR)'/pdo_$*.dockerfile .
 
+build_sgx_services : repository
+	docker build $(DOCKER_ARGS) \
+        --build-arg PDO_VERSION=$(PDO_VERSION) \
+        --build-arg SGX_MODE=HW \
+        --tag pdo_services:$(PDO_VERSION) \
+        --file $(DOCKER_DIR)/pdo_services.dockerfile .
+
 # docker build dependencies
 build_client: build_base
 build_services: build_services_base
 build_services_base: build_base
 build_ccf: build_ccf_base
 
+build_sgx_services: build_services_base
```

Comment:
Nay unavoidable reason to repeat target rather than combine this with the one above? Otherwise having a single target is much clearer ...

Reply:
You're right, as there is a single target now, it's clearer to consolidate these -- I have updated the code.